pkg/apis/controlplane/types.go

```diff
@@ -340,6 +340,16 @@ type NodeStatsSummary struct {
 	AntreaClusterNetworkPolicies []NetworkPolicyStats
 	// The TrafficStats of Antrea NetworkPolicies collected from the Node.
 	AntreaNetworkPolicies []NetworkPolicyStats
+	// The stats related to multicast collected from the Node.
```

Comment:
Revise the comments too: The stats ... -> Multicast group information

Reply:
Updated. Thanks